src/Microsoft.DotNet.Build.Tasks/Targets/packages.targets

```diff
@@ -18,6 +18,10 @@
     <NuGetOutputDirectory>&quot;$(PackagesOutDir.TrimEnd('\'))&quot;</NuGetOutputDirectory>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <LocalPublishPropsFile Condition="'$(LocalPublishPropsFile)'==''">$(SourceDir)\localpublish.props</LocalPublishPropsFile>
```

Comment:
If we don't want this to be $(SourceDir), what should it be? $(BinDir)? Something else?